gozw.go

```diff
@@ -254,6 +286,7 @@ func (c *Client) Shutdown() error {
 
 func (c *Client) FactoryReset() error {
 	c.serialAPI.FactoryReset()
+	c.PruneNodes()
```

Comment:
c.PruneNodes returns an error, but it's ignored. It's probably best to check for and pass this back up the stack